lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -263,7 +263,7 @@ class SetUndoableCommand : public MayaUsd::ufe::UsdUndoableCommand<Ufe::Undoable
         MayaUsd::ufe::InSetAttribute inSetAttr;
         MayaUsd::ufe::UsdUndoableCommand<Ufe::UndoableCommand>::undo();
     }
-    
+
```

Comment:
clang-format run on UsdAttribute.cpp causing linter check failure

Reply:
Yes another dev also had this same problem and has fixed it in their pending PR.